vmaas_sync/metrics_cyndi.go

```diff
@@ -0,0 +1,125 @@
+package vmaas_sync //nolint:golint,stylecheck
+
+import (
+	"app/base/database"
+	"app/base/models"
+	"app/base/utils"
+	"fmt"
+	"github.com/jinzhu/gorm"
+	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
+	"time"
+)
+
+const (
+	allSystemCount        = "allSystem"
+	uniqueTagsCount       = "uniqueTags"
+	systemsSapSystemCount = "systemsSapSystem"
+	systemsWithTagsCount  = "systemsWithTag"
+)
+
+var (
+	cyndiSystemsCnt = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "patchman_engine",
+		Subsystem: "vmaas_sync",
+		Name:      "cyndi_systems",
+	}, []string{"type"})
+
+	cyndiTagsCnt = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "patchman_engine",
+		Subsystem: "vmaas_sync",
+		Name:      "cyndi_tags_count",
+	}, []string{"type"})
+
+	tagsPerSystem = prometheus.NewHistogram(prometheus.HistogramOpts{
```

Comment:
Which histogram bins are used here. Are they appropriate for our usage? I would be good to discuss with someone from inventory/cyndi which values we can expect here to choose good resolution.

Reply:
Ok, we can remove this metric for now and add it later. Let's get this one done asap.